src/mutual_information.jl

```diff
@@ -82,6 +71,47 @@ function bitinformation(A::AbstractArray{T};
     return mutual_information(A1view,A2view;kwargs...)
 end
 
+"""
+    M = bitinformation(A::AbstractArray{T}, mask::BitArray) where {T<:Union{Integer,AbstractFloat}}
+
+Bitwise real information content of array `A` calculated from the bitwise mutual information
+in adjacent entries in A along dimension `dim` (optional keyword). Array `A` is masked through
+trues in entries of the mask `mask`. Masked elements are ignored in the bitwise information calculation."""
+function bitinformation(A::AbstractArray{T},
+                        mask::BitArray;
```

Comment:
would it be possible for `bitinformation` to guess the mask based on a masking value, such as `-9.e+33`?

Or I'd appreciate a short help to how to create `mask`. What the new julia user tries and fails is `mask = ncfile.vars[varnames[1]][:,:,:,:] == -9.e+33`

Reply:
While it would be possible to guess the mask, I don't want to do that, as for the moment the number format `T` can be any bitstype. Hence making the assumption that -9e33 is a mask may work well in one format (e.g. Float64) but not necessarily in others (integer, posits, etc., or a NaN in Float16 isn't necessarily a NaN in BFloat16 etc.). We could define `bitinformation(A::AbstractArray{T},mask::T)` which creates a mask based on the bit pattern in the scalar `mask` if that would be helpful. In general, Julia creates a BitArray for any broadcasted comparison, e.g.
```julia
julia> A = rand(3,3)
3×3 Matrix{Float64}:
 0.119336  0.127636  0.808542
 0.439229  0.388266  0.52312
 0.899243  0.992992  0.549393

julia> A .< 0.5
3×3 BitMatrix:
 1  1  0
 1  1  0
 0  0  0
```
what you are missing is the dot, i.e. `.==` instead of `==` broadcasting in Julia is a bit more conservative than in python...